packages/chat-headless/src/ChatHeadless.ts

```diff
@@ -28,9 +30,30 @@ export class ChatHeadless {
   private chatCore: ChatCore;
   private stateManager: ReduxStateManager;
 
-  constructor(config: ChatConfig) {
+  /**
+   * Constructs a new instance of the {@link ChatHeadless} class.
+   *
+   * @public
+   *
+   * @param config - The configuration for the {@link ChatHeadless} instance
+   * @param saveToSessionStorage - Whether to save the instance's {@link ConversationState} to session storage
```

Comment:
also add that this will default to true. rerun build to update the generated doc after